yt/fields/fluid_fields.py

```diff
@@ -195,12 +195,29 @@ def grad_func(axi, ax):
         slice_3dl = slice_3d[:axi] + (sl_left,) + slice_3d[axi+1:]
         slice_3dr = slice_3d[:axi] + (sl_right,) + slice_3d[axi+1:]
         def func(field, data):
-            ds = div_fac * data[ftype, "d%s" % ax]
```

Comment:
What happened to `div_fac`? I don't see it in the code below. Is it no longer necessary?